README.md

```diff
@@ -111,15 +114,22 @@ data:
 }
 ```
 
-### How to intrpret in code:
+### How to interpret:
  
-A piece of software reading this could:  
+Starting from the time series variables:
 1) See CF-1.8 conventions  
-2) See the timeSeries featureType  
-3) Find the timeseries\_id cf\_role  
-4) Find the coordinates  
-5) See that the coordinates have a standard\_name `polygon x node` and `ploygon y node` to determine that these are polygons according to this new specification.  
-6) Find the variable with geom_coordinates that point to the nodes.
-7) Find the variable with `continuous_ragged_dimension` to determine how to index into the coordinate index.
-7) Iterate over polygons, picking out geometries for each `timeSeries` using the count to pick out the required nodes.  
-
+2) See the `timeSeries` featureType  
+3) Find the `timeseries_id` `cf_role`  
+4) Find the `coordinates` attribute of data variables. 
+5) See that the variables indicated by the `coordinates` attribute have a `standard_name` `polygon x node` and `ploygon y node` to determine that these are polygons according to this new specification.  
```

Comment:
I agree with @twhiteaker. I think we want to stick with generic `geometry_*_node` instead of `point/polygon/line_*_node`. The coordinate index variable holds everything necessary to interpret the node variables. I am guessing multiple variables with the same standard names are acceptable.